portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -16,17 +16,36 @@
 
 import java.io.IOException;
 import javax.servlet.annotation.WebServlet;
+import java.util.ArrayList;
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
+import com.google.gson.Gson;
 
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
 
-  @Override
-  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    response.setContentType("text/html;");
-    response.getWriter().println("<h1>Hello world!</h1>");
-  }
+    private String name;
+
+    private ArrayList<String> info = new ArrayList<String>() {{
+        add("My name is Jordan");
+        add("I hope to speak russian fluently one day");
+        add("I want to have a nice life.");
+    }};
+    @Override public void init() {
+        name = "Jordan";
```

Comment:
Could this be initialized where the field is declared?

Reply:
Sure